packages/components/molecules/f-tabs/src/components/Tabs.vue

```diff
@@ -1,46 +1,142 @@
-import tenantConfigs from '../tenants';
+
+const DIRECTION = {
+    LEFT: 'LEFT',
+    RIGHT: 'RIGHT'
+};
 
 export default {
-    name: 'VueTabs',
-    components: {},
+    name: 'Tabs',
     props: {
         locale: {
             type: String,
             default: ''
+        },
+        animate: {
+            type: Boolean,
+            default: false
         }
     },
-    data () {
-        const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
-        const localeConfig = tenantConfigs[locale];
-
+    data: () => ({
+        direction: null,
+        activeTab: '',
+        tabs: []
+    }),
+    provide () {
+        const component = this;
+        const tabsComponent = {};
+        Object.defineProperty(tabsComponent, 'activeTab', {
+            enumerable: true,
+            get: () => this.activeTab
+        });
+        Object.defineProperty(tabsComponent, 'animationDirection', {
+            enumerable: true,
+            get: () => this.direction
+        });
+        Object.defineProperty(tabsComponent, 'animate', {
+            enumerable: true,
+            get: () => this.animate
+        });
         return {
-            copy: { ...localeConfig }
+            register (tab) {
+                component.addTab(tab);
+            },
+            tabsComponent
         };
+    },
+    methods: {
+        selectTabIndex (name) {
+            const previousIndex = this.tabs.findIndex(t => t.name === this.activeTab);
+            const newIndex = this.tabs.findIndex(t => t.name === name);
+            if (newIndex > previousIndex) {
+                this.direction = DIRECTION.RIGHT;
+            } else {
+                this.direction = DIRECTION.LEFT;
+            }
+            this.activeTab = name;
+        },
+        addTab (tab) {
+            if (tab.selected) {
+                this.activeTab = tab.name;
+            }
+            this.tabs = [...this.tabs, tab];
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
 
+$tabs-link-colour         : $grey--darkest;
+$tabs-link-font-weight    : $font-weight-bold;
+$tabs-link-border-colour  : $brand--orange;
+
 .c-tabs {
+    width: 100%;
+    height: 100%;
+}
+.c-tabs-header {
+    width: 100%;
+    height: 42px;
+}
+.c-tabs-buttons {
     display: flex;
-    justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid $red;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
+    align-items: center;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    height: 100%;
+
+    @include media('<narrowMid') {
+        width: 100%;
+    }
+
+}
+.c-tabs-button {
+    padding: spacing(x1.5) spacing(x2);
+    cursor: pointer;
+    color: $tabs-link-colour;
+    border: none;
+    background-color: transparent;
+    outline: none;
+
+    @include media('<narrowMid') {
+        /* stylelint-disable */
+        flex: 1 1 0px;// important px for IE
```

Comment:
Why's this important for IE? Does the flex styling not work otherwise?

Reply:
IE has a problem parsing the flex property. For some reason it can not handle the 0 without a unit apparently, something I have been made aware of.